mungegithub/mungers/inactive-review-handler.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	githubapi "github.com/google/go-github/github"
+	"github.com/spf13/cobra"
+	"k8s.io/test-infra/mungegithub/features"
+	"k8s.io/test-infra/mungegithub/github"
+	"k8s.io/test-infra/mungegithub/mungers/matchers"
+	"time"
+)
+
+const (
+	msgHeader   = "Sorry it is taking a long time for @%s to review your PR. This person may be on vacation or otherwise occupied. "
```

Comment:
Being more explicit.  I think it may be better NOT to mention the person by name directly.  Passive voice is okay here.  

`Sorry the review process for your PR has stalled.   Your reviewer(s) may be on vacation or otherwise occupied.  Consider unassigning them using the /unassign [command](https://github.com/kubernetes/test-infra/blob/master/commands.md) and assigning %s`